Update README.md and fix add_modules.py about module maintainer

meteorcloudy opened the pull request:
The Bazel team should not be the default maintainer of a module, if not set, the field will just be empty in the generated json file.

meteorcloudy commented:
FYI @alexeagle

philwo commented:
I think we should add an explanatory message about what a "Maintainer" is, for example to avoid people just adding the project owners without their consent or adding themselves without knowing what this actually means. I don't have time to draft something up today though, sorry :/

meteorcloudy commented:
@philwo I updated the README.md file and added a link to the `module maintainer` section from the script. Please help rephrase it if necessary.

@alexeagle I tried to clarify the confusion of `module maintainer` and `presubmit.yml` you raised up in the README.md file. Please also take a look.

philwo commented on README.md:
```diff
@@ -3,13 +3,50 @@
+
+### Module Maintainer
+
+A person who maintains a speicify module entry in this registry is a module maintainer. The name and contact information of the module maintainers are recorded in the metadata.json file for each module.
```
typo: specific

meteorcloudy replied:
Thanks!

philwo commented on README.md:
```diff
@@ -3,13 +3,50 @@
+
+### Registry Maintainer
+
+Currently, the Bazel team is the registry maintainer, who are repsonsible for:
```
typo: responsible

meteorcloudy replied:
Done

philwo reviewed (approved):
Great, thank you!
